src/main/java/dev/walshy/sfmetrics/charts/NewServersChart.java

```diff
@@ -0,0 +1,42 @@
+package dev.walshy.sfmetrics.charts;
+
+import org.bstats.bukkit.Metrics.SingleLineChart;
+import org.bukkit.Server;
+import org.bukkit.plugin.Plugin;
+
+import dev.walshy.sfmetrics.VersionDependentChart;
+import io.github.thebusybiscuit.slimefun4.api.SlimefunBranch;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+
+/**
+ * This single line graph shows the amount of {@link Server Servers} that installed
+ * Slimefun for the very first time at that moment in time.
```

Comment:
Maybe we should indicate it's until the server is stopped/restarted. "at that moment in time" to me indicates that this would only be recorded once and not keep being recorded until the server was stopped.